gallery/src/PageHeader/HostedVersionsSelect.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+import { without } from 'ramda';
+import { NxStatefulDropdown } from '@sonatype/react-shared-components';
+import compareVersions from 'compare-versions';
+
+import packageJson from '../../package.json';
+
+import './HostedVersionsSelect.scss';
+
+const CURRENT_VERSION = packageJson.version;
+const HOSTED_VERSIONS_JSON_PATH = '/hosted-versions.json';
+
+// eslint-disable-next-line no-undef
+const PRODUCTION = process.env.PRODUCTION;
+
+const getHostedVersions = async () => {
+  const response = await fetch(HOSTED_VERSIONS_JSON_PATH);
+
+  if (!response.ok) {
+    throw new Error('Failed to load hosted versions.');
+  }
+
+  return response.json();
+};
+
+const HostedVersionsSelect = () => {
+  const [isLoadingHostedVersions, setIsLoadingHostedVersions] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const [hostedVersions, setHostedVersions] = useState([CURRENT_VERSION]);
+
+  useEffect(() => {
+    setIsLoadingHostedVersions(true);
+
+    const loadHostedVersions = async () => {
+      try {
+        const hostedVersions = await getHostedVersions();
+        const sortedVersions = without(['latest'], hostedVersions).sort(compareVersions).reverse();
```

Comment:
Minor: slightly more efficient
```suggestion
        const sortedVersions = without(['latest'], hostedVersions).sort(pipe(compareVersions, negate));
```